packages/perseus/src/widgets/label-image/marker.tsx

```diff
@@ -29,6 +29,11 @@ type State = {
     isFocused: boolean;
 };
 
+function shouldReduceMotion(): boolean {
+    const mediaQuery = window.matchMedia("(prefers-reduced-motion: reduce)");
+    return !mediaQuery || mediaQuery.matches;
```

Comment:
Why default to reduced motion when no media query is found?

Reply:
I see. That does make sense to me, but I would like it if we had confirmation from Sarah B. I'll approve and humbly request you run this choice by our UX designer. Thanks Nicole!